.github/scripts/end2end/deploy-zenko.sh

```diff
@@ -43,6 +44,14 @@ else
     export ZENKO_MONGODB_ENDPOINT="dev-db-mongodb-primary-0.dev-db-mongodb-headless.default.svc.cluster.local:27017"
 fi
 
+if [ -n "${ZENKO_ANNOTATIONS}" && ${TIME_PROGRESSION_FACTOR} != 1]; then
+    export ZENKO_ANNOTATIONS="${ZENKO_ANNOTATIONS}
+    zenko.io/time-progression-factor: \"${TIME_PROGRESSION_FACTOR}\""
+elif [ ${TIME_PROGRESSION_FACTOR} != 1 ]; then
+    export ZENKO_ANNOTATIONS="annotations:
+    zenko.io/time-progression-factor: \"${TIME_PROGRESSION_FACTOR}\""
+fi
+
```

Comment:
* `&&` separates two commands: to have multiple tests, must instead either use `-a` (`if [ cond1 -a cond2 ]; then`) or mulitple "brackets" blocks (`if [ cond1 ] && [ cond2 ] ; then`)
* avoid repetition by using [variable expansion with default value](https://www.debuntu.org/how-to-bash-parameter-expansion-and-default-values/)
* may be safer (as in "future-proof") to also handle the case where TIME_PROGRESSION_FACTOR is not set

```suggestion
if [  -n ${TIME_PROGRESSION_FACTOR} -a ${TIME_PROGRESSION_FACTOR} != 1 ]; then
    export ZENKO_ANNOTATIONS="${ZENKO_ANNOTATIONS:-annotations:}
    zenko.io/time-progression-factor: \"${TIME_PROGRESSION_FACTOR}\""
fi
```